controllers/halayerset_controller_test.go

```diff
@@ -15,13 +15,15 @@ import (
 )
 
 const (
-	defaultNamespace       = "default"
-	haLayerCRName          = "test"
-	node1Name              = "mock-cluster1-node"
-	node2Name              = "mock-cluster2-node"
-	originalFenceAgentName = "mock-fence-org"
-	fenceAgentType         = "fence_mock"
-	updatedFenceAgentName  = "mock-fence-changed"
+	defaultNamespace      = "default"
```

Comment:
Some consts look the same, like you have deleted and inserted the same consts, e.g., defaultNamespace, haLayerCRName, and etc.
Why do we see a change, difference, if there is no difference. Or there is a change?

Reply:
Location change will show you delete at original location and addition at the new location

Reply:
I know, it looks weird because they have the same line number and order... My guess is that there is an extra space before or after